test/e2e/lifecycle/bootstrap/util.go

```diff
@@ -137,6 +145,7 @@ func WaitForBootstrapTokenSecretToDisappear(c clientset.Interface, tokenID strin
 	})
 }
 
+// WaitForBootstrapTokenSecretNotDisappear waits for bootstrap token secret not to be disappeared
```

Comment:
This function takes time for the specified timeout as success path.
it is nice to describe it at the above.

Reply:
WaitForBootstrapTokenSecretNotDisappear waits for bootstrap token secret not to be disappeared 
=》
WaitForBootstrapTokenSecretNotDisappear takes time for the specified timeout as success path.
ok?/cc @oomichi

Reply:
How about
```
WaitForBootstrapTokenSecretNotDisappear waits for bootstrap token secret not to be disappeared and takes time for the specified timeout as success path.
```
?

Reply:
both.Sure Thanks